Storage/container/many_to_many.h

```diff
@@ -105,12 +123,27 @@ class GenericManyToMany {
     return operator[](std::make_pair(row, col));
   }
 
+  ImmutableOptional<std::chrono::microseconds> LastModified(const key_t& key) const {
```

Comment:
Above the very method, along the lines of `Pass the key by a const reference, as `key_t` is an `std::pair<row_t, col_t>`..